example_project/example_project/settings.py

```diff
@@ -53,34 +53,37 @@
 
 WSGI_APPLICATION = 'example_project.wsgi.application'
 
+# Internationalization
+# https://docs.djangoproject.com/en/1.6/topics/i18n/
```

Comment:
Oops, sorry. Just saw, that the related lines are in https://github.com/PolyPasswordHasher/PolyPasswordHasher-Django/pull/9/files#diff-3684049e89acfc4131717dc0718de375R87.
Consider updating the link to
```
# https://docs.djangoproject.com/en/1.10/topics/i18n/
```
and move the comments down!